lib/fixme.ex

```diff
@@ -2,20 +2,26 @@ defmodule FIXME do
   defmacro fixme(
     {:-, _, [{:-, _, [year, month]}, day]},
     message,
-    opts \\ :default_opts
+    opts \\ []
   ) do
     {opts, _} = Code.eval_quoted(opts)
 
     # Injectable "today" for tests.
-    current_date = case opts do
-      :default_opts -> :calendar.local_time |> elem(0)
-      [today: date] -> date
-    end
+    current_date = opts[:today] || (:calendar.local_time |> elem(0))
+
+    # Opt into compile time warnings
+    warn = opts[:warn]
```

Comment:
@henrik I have added the option in config, the existing option when provided to the macro call can now be used to override the settings provided in the config. Please have a look and let me know if any further changes are required.